content/hi/docs/reference/glossary/pod-disruption.md

```diff
@@ -0,0 +1,22 @@
+---
+title: पोड डिसृपशन (Pod Disruption)
```

Comment:
डिसरप्सन  will be more suitable

Reply:
Should i change it to डिसरप्सन? it seems डिसृपशन sounds more better. But if you will confirm , i will change it to डिसरप्सन.

Reply:
डिसृपशन is correct

Reply:
Ok, thank you for the confirmation.

Reply:
Should we translate disruption to व्यवधान? @Garima-Negi @verma-kunal @anubha-v-ardhan ? Also, there is a small spelling nit to be addressed below, @Aayush987 .

Reply:
I feel that if we can translating disruption, we could also use `विघटन`. Thoughts, WDYT @divya-mohan0209 @Garima-Negi @anubha-v-ardhan @verma-kunal ?

Reply:
SGTM.

Reply:
Should i change it to `विघटन` then? @divya-mohan0209

Reply:
So sorry for the delay, I will do the changes by tomorrow.

Reply:
Done the changes.